src/Network/Network/AzureFirewallPolicy/FilterRule/NewAzureFirewallPolicyFilterRuleCollectionCommand.cs

```diff
@@ -62,7 +62,7 @@ public override void Execute()
                 Priority = this.Priority,
                 Rules = this.Rule?.ToList(),
                 Action = new PSAzureFirewallPolicyFilterRuleCollectionAction { Type = ActionType },
-                RuleCollectionType = "FirewallPolicyFilterRule"
+                RuleCollectionType = "FirewallPolicyFilterRuleCollection"
```

Comment:
are these two rule collection type equivalent?

Reply:
Yes this is just a name change.